test/hotspot/jtreg/runtime/cds/appcds/StaticArchiveWithLambda.java

```diff
@@ -52,7 +52,8 @@ public static void main(String[] args) throws Exception {
                        "-cp", appJar,
                        "-Xlog:class+load,cds")
             .setArchiveName(archiveName);
-        CDSTestUtils.createArchiveAndCheck(opts);
+        CDSTestUtils.createArchiveAndCheck(opts)
+            .shouldContain("Skipping java/lang/invoke/BoundMethodHandle$Species_LLLL because it is dynamically generated");
```

Comment:
```suggestion
        CDSTestUtils.createArchiveAndCheck(opts)
                    .shouldContain("Skipping java/lang/invoke/BoundMethodHandle$Species_LLLL because it is dynamically generated");
```